api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -28,6 +28,10 @@ type DRState string
 
 // These are the valid values for DRState
 const (
+	// WaitForUser, state recorded in DRPC status to indicate that we are
+	// waiting for the user to take an action after hub recover.
+	WaitForUser = DRState("WaitForUser")
```

Comment:
How ramen knows that there was a hub recovery? what is the difference between a hub recovery and a new install?

Reply:
Ramen does not know. Trying to rebuild its state based on what is out there.